pkgs/by-name/ne/nekoray/package.nix

```diff
@@ -157,6 +166,5 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.gpl3Plus;
     mainProgram = "nekoray";
     maintainers = with lib.maintainers; [ tomasajt ];
-    platforms = lib.platforms.linux;
```

Comment:
I think instead of being implicit about the platforms, we should do lib.platforms.linux ++ lib.platforms.darwin